src/solver/utils/mps_utils.h

```diff
@@ -28,39 +28,52 @@ void OPT_EcrireJeuDeDonneesLineaireAuFormatMPS(void*,
 class I_MPS_writer
 {
 public:
-    I_MPS_writer() = default;
+    I_MPS_writer(int year, int week, int currentOptimNumber) : 
+       year_(year), week_(week), current_optim_number_(currentOptimNumber)
+    {}
+    I_MPS_writer()
+    {}
```

Comment:
```suggestion
    I_MPS_writer(int year, int week, int currentOptimNumber) : 
       year_(year), week_(week), current_optim_number_(currentOptimNumber) = default;
    I_MPS_writer() = default;
```

Reply:
Having a **default** constructor with an initialization list does not compile : 
```C++
I_MPS_writer(int year, int week, int currentOptimNumber) : 
       year_(year), week_(week), current_optim_number_(currentOptimNumber) = default;
```

Reply:
**default** identifier after initialization list does not compile